README.md

```diff
@@ -83,7 +83,7 @@ stateDiagram-v2
 
 ### Earning Power Calculator
 
-The earning power calculator determines which stakers are eligible for a reward. This implementation uses an oracle. An oracle is needed because eligibility depends on off-chain behavior.
+The earning power calculator determines which stakers are eligible for a reward, based on their stake and their governance delegatee. The calculator is a modular component of the staker, which can be customized and updated by owner of the Staker, such as a DAO. One provided implementation uses an oracle. An oracle is needed because eligibility depends on the off-chain behavior of DAO delegates.
```

Comment:
It might be more accurate to indicate that beyond eligibility the earning power calculator also determines the proportion of rewards a staker receives. Also, I find using depositor over staker to be clearer as stakers could also refer to multiple Staker systems.